internal/app/config.go

```diff
@@ -101,6 +101,20 @@ func (a *App) LoadConfiguration() error {
 	return nil
 }
 
+func (a *App) OidcEnabled() bool {
+	if a.v.GetBool("oidc.enabled") {
+		if len(a.Config.APIServerJWTAuth) == 0 {
+			a.Logger.Fatal("OIDC enabled without configuration")
+		}
+		a.Logger.Info("OIDC enabled")
+
+		return true
+	}
+
+	a.Logger.Info("OIDC disabled")
```

Comment:
Super nit pick.

Should OIDC being disabled be a warning?

Reply:
Not really, OIDC availability is deployment specific and this is informational so we can go check if its enabled or disabled from the logs.